src/common/ErrorMessage.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ErrorMessage = (props) => {
+  const { message, retry, param } = props;
+  return (
+    <div className='error-message'>
+      <img src='https://goo.gl/1VpMZd' alt='error-icon' />
```

Comment:
Can we have such images as constants outside the component and then import them where necessary? Especially moving forward.

Reply:
Okay, that can be done.